server/src/actors/store.rs

```diff
@@ -176,7 +176,6 @@ struct AttachmentKafkaMessage {
     /// The project id for the current event.
     project_id: ProjectId,
     /// The attachment.
-    #[serde(flatten)]
     attachment: ChunkedAttachment,
```

Comment:
This needs a change in the Sentry consumer.

Reply:
The sentry consumer is yet to be implemented for individual attachments so that should be fine

Reply:
Right! I thought the message was already typed out somewhere. How sarcastic.